blog/_posts/2021-11-04-docker-build-alternatives.md

```diff
@@ -0,0 +1,234 @@
+---
+slug: "docker-build-alternatives"
+date: 2021-11-03
+author: nick
+layout: blog
+title: "How to Cope When Docker Builds are Slowing You Down"
```

Comment:
I feel like "Docker builds slowing you down" restricts the problem space unnecessarily.
Instead, I'd suggest something like "When Docker Builds are not right for you" (or something better)

Reply:
Ya, i was trying to do a hat-tip to Jason's tweet below. But I think it needs a catchier title. let me bat some ideas around